packages/react/src/components/TreeView/TreeView-test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { Document16, Folder16 } from '@carbon/icons-react';
+import { settings } from 'carbon-components';
+import TreeView, { TreeNode } from './';
+
+const { prefix } = settings;
+
+describe('TreeView', () => {
```

Comment:
Seems like there is a ton of great support for keyboard interaction, should we add tests for that as well?

Reply:
I chose to omit those tests for now until I am certain about the keyboard interaction patterns in the component (which is partly why this is an unstable component)